main.go

```diff
@@ -88,6 +90,83 @@ func (e *errHealthCheck) Error() string {
 	return "request to url " + e.URL + " returned invalid status code " + strconv.Itoa(e.StatusCode) + " after " + strconv.Itoa(e.Tries) + " tries"
 }
 
+// version is the published version of the utility
+var version string
+
+const (
+	// SchemesFlag is the Schemes Flag
```

Comment:
Why did you say the strings need to be constants again?

Reply:
This becomes a problem with flags like `verbose` where you end up doing `verbose := v.GetString("verbose")` in a bunch of places. While contrived it's better to do `verbose := v.GetString(VerboseFlag)` because you're unlikely to mispell `"verbose"` somewhere. I also see it's helpful in error messages like `return fmt.Errorf("%s flag is misconfigured", VerboseFlag)`. And then when you want to change that flag from `--verbose` to `--debug-logging` you only have to change the flag name in one place and it works. Does that help?